app/controllers/peer_reviews_controller.rb

```diff
@@ -133,17 +138,19 @@ def assign(reviewer_groups, reviewee_groups)
   def unassign(selected_reviewee_group_ids, reviewers_to_remove_from_reviewees_map)
     # First do specific unassigning.
     reviewers_to_remove_from_reviewees_map.each do |reviewee_id, reviewer_id_to_bool|
-      reviewer_id_to_bool.each do |reviewer_id, dummy_value|
+      reviewer_id_to_bool.each do |reviewer_id, _|
         reviewee_group = Grouping.find_by_id(reviewee_id)
         reviewer_group = Grouping.find_by_id(reviewer_id)
         peer_review = reviewer_group.review_for(reviewee_group)
-        peer_review.destroy
+        if peer_review
+          peer_review.destroy
```

Comment:
Use `peer_review&.destroy` (this is the [safe navigation operator](http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/)).